### PR TITLE
Share sheet improvements (internal sharing)

### PR DESCRIPTION
**Pull request by ChrisEdS**
## Description
Share Sheet for internal shares was cleaned up a bit. Unfortunately, only CSS can be used in this area.

This PR contains:
- Alignment of the icons has changed
- Spacing between the icons has been increased
- Background color and dividing line for the individual shares
- The user name is truncated to prevent a new line

## Related Issue

- Fixes https://github.com/owncloud/enterprise/issues/3979

## Motivation and Context
The share sheet seemed a bit untidy. With my code changes, it seems calmer.

## How Has This Been Tested?

- Manually tested in Firefox and Chrome

## Screenshots:
| Before  | After  |  
|---|---|
| image  |  image |

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by micbar** [comment]
LGTM, please add changelog https://handbook.owncloud.com/changelog_entries.html#creating-changelog-entries

**Review by pmaier1 — APPROVED** [review]
Thx :+1: (not a technical review :) )

**Comment by phil-davis** [comment]
An example with long usernames and/or display name:
Before:
Long-Names-Before
After:
Long-Names-After

**Comment by ChrisEdS** [comment]
@pmaier I can revert the changes for:
- The user name is truncated to prevent a new line

The max length will be 256px then.

image

File: changelog/unreleased/37491 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Change: Share sheet improvements (internal sharing)
+
+Share Sheet for internal shares was cleaned up a bit.
+
+- Alignment of the icons has changed
+- Spacing between the icons has been increased
+- Background color and dividing line for the individual shares
+- The user name is truncated to prevent a new line
```
Comment (mmattel) [inline changelog/unreleased/37491:8]: Truncate of user name has been reverted with your commit 8e7736f, therefore pls remove.

Comment (phil-davis) [reply changelog/unreleased/37491]: I removed the line, and squashed it into that commit, and rebased. CI is running again.
Now I will do some manual testing to see what it looks like.

**Pull request merged**